posthog/migrations/0410_action_steps_population.py

```diff
@@ -0,0 +1,51 @@
+# Generated by Django 4.2.11 on 2024-05-03 08:27
+
+from django.db import migrations
+from django.core.paginator import Paginator
+
+
+def copy_action_steps_to_json(apps, schema_editor):
+    from posthog.models import Action
+
+    all_actions_with_steps = Action.objects.prefetch_related("action_steps").order_by("pk").all()
+
+    paginator = Paginator(all_actions_with_steps, 100)
+
+    for page_number in paginator.page_range:
+        page = paginator.page(page_number)
+
+        print(f"Processing page {page_number} of {paginator.num_pages}")
```

Comment:
should do struct log logging here or some counter into prom instead of the print

(or just delete the print :))